PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -17,11 +17,27 @@ public static void Main(string[] args)
             }
         }
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
+        Console.WriteLine("Your character match is: " + UserCharacterResult(numberCorrect,questions.Length) + "!");
     }
 
-    public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+    public static string GetPercentCorrect(double numberCorrectAnswers, double numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return Math.Round((numberCorrectAnswers / numberOfQuestions * 100),2) + "%";
```

Comment:
```suggestion
        return ((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
If used I would recommend reverting the method variable types to integers.